packages/shared/src/components/modals/MarketingCTAModal.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { ReactElement } from 'react';
+import { Modal, ModalProps } from './common/Modal';
+import { ButtonSize } from '../buttons/Button';
+import type { MarketingCTA } from '../cards';
+import { CTAButton, Description, Header, Title } from '../cards';
+import { CardCover } from '../cards/common/CardCover';
+
+export interface NewRankModalProps extends ModalProps {
+  marketingCTA: MarketingCTA;
+}
+export const MarketingCTAModal = ({
+  marketingCTA,
+  onRequestClose,
+  ...modalProps
+}: NewRankModalProps): ReactElement => {
+  const { tagColor, tagText, title, description, image, ctaUrl, ctaText } =
+    marketingCTA;
+
+  const onModalClose: typeof onRequestClose = (param) => {
+    onRequestClose(param);
+  };
+
+  return (
+    <Modal
+      {...modalProps}
+      kind={Modal.Kind.FlexibleCenter}
+      size={Modal.Size.Small}
+      onRequestClose={onModalClose}
+    >
+      <div className="p-6 !pt-4">
+        {tagColor && tagText && (
+          <Header
+            onClose={onModalClose}
+            tagColor={tagColor}
+            tagText={tagText}
+            buttonSize={ButtonSize.Medium}
+          />
+        )}
+        <Title className="!typo-large-title">{title}</Title>
+        {description && (
+          <Description className="!typo-body">{description}</Description>
```

Comment:
Wish they could get props like button does for sizes for example

Reply:
Yeah they actually will with some more components.
In LeanFeed we have something like this, for now it was easier to do as such (because of all deps)